src/DataStructures.jl

```diff
@@ -5,6 +5,10 @@ module DataStructures
                 isbitsunion, isiterable, dict_with_eltype, KeySet, Callable, _tablesz,
                 findnextnot, unsafe_getindex, unsafe_setindex!, peek
 
+    if !isdefined(Base, :popat!)
+        export popat!
+    end
```

Comment:
What do you think of:
```suggestion
    export popat!  # Export for old version of julia where Base doesn't export this
```

I think it should work since it is either exporting something new (fine)
or it is a no-op by exporting something already exported